File: src/main/java/akko/ddbot/task/LiverInit.java
package akko.ddbot.task;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.util.LinkedList;
import java.util.List;

public class LiverInit {
    public static List<String> liverList;

    public void init() {
        liverList = new LinkedList<>();
        
        try 
        {
            Class.forName("org.sqlite.JDBC");
            Connection sqliteC = DriverManager.getConnection("jdbc:sqlite:db/GroupInfo.db");
            ResultSet set = sqliteC.createStatement().executeQuery("select * from vLiver;");
            
            while (set.next()) {
                liverList.add(set.getString("vID"));
            }
            sqliteC.close();
        } 
        catch (SQLException | ClassNotFoundException e) 
        {
            e.printStackTrace();
        }
    }
}

File: src/main/java/akko/ddbot/task/LiverInit.kt
package akko.ddbot.task

import akko.ddbot.BotMainActivity
import akko.ddbot.sql.SQLFun
import java.sql.Connection
import java.sql.ResultSet
import java.sql.SQLException
import java.util.*

class LiverInit {
    companion object {
        var liverList: MutableList<String>? = null
        fun init() {
            liverList = LinkedList()
            try {
                val pair = SQLFun().executeQuery("SELECT * FROM groupinfo.vliver;")
                val set: ResultSet = pair!!.first
                do {
                    (liverList as LinkedList<String>).add(set.getString("vID"))
                } while (set.next())
                pair.second.close()
            } catch (e: SQLException) {
                BotMainActivity.ExceptionLogger!!.debug(e.message)
                e.printStackTrace()
            }
        }
    }
}